### PR TITLE
BUCash release notes for version 1.5.1.0 Relese Candidate 1

### PR DESCRIPTION
**Pull request by sickpig**

### CODE REVIEW

File: doc/release-notes/release-notes-bucash1.5.1.0-rc1.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+This is a main release candidate based of Bitcoin Unlimited compatible
+with the Bitcoin Cash specifications you could find here:
+
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/uahf-technical-spec.md (Aug 1st '17 Protocol Upgrade, bucash 1.1.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/nov-13-hardfork-spec.md (Nov 13th '17 Protocol Upgrade, bucash 1.1.2.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/may-2018-hardfork.md (May 15th '18 Protocol Upgrade, bucash 1.3.0.0, 1.3.0.1, 1.4.0.0)
+- https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md (Nov 15th '18 Protocol Upgrade, bucash 1.5.0.0, 1.5.0.1, 1.5.0.2)
+
+This release is compatible with both [Bitcoin Cash](https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2018-nov-upgrade.md) and [SV](https://github.com/bitcoin-sv/bitcoin-sv/blob/master/doc/release-notes.md) changes to the consensus rules.
```
Comment (awemany) [inline doc/release-notes/release-notes-bucash1.5.1.0-rc1.md]: I would reword this to 'This release candidate is compatible with ...' as well.

**Comment by awemany** [comment]
Typo in commit message: "Relese". Minor nit above. Otherwise looks good. Thank you! :)

File: doc/release-notes/release-notes-bucash1.5.1.0-rc1.md (hunk 1 above)
Comment (sickpig) [reply doc/release-notes/release-notes-bucash1.5.1.0-rc1.md]: fixed

**Comment by sickpig** [comment]
@awemany thanks for the feedback, got them fixed.

**Comment by Greg-Griffith** [comment]
We should remove the downgrade section because downgrading to older clients will put you on minority/dead forks.

File: doc/release-notes/release-notes-bucash1.5.1.0-rc1.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+If you are running an older version, shut it down. Wait until it has completely
+shut down (which might take a few minutes for older versions), then run the
+installer (on Windows) or just copy over /Applications/Bitcoin-Qt (on Mac) or
+bitcoind/bitcoin-qt (on Linux).
+
+If you are upgrading from a release older than 1.1.2.0, your UTXO database will be converted
+to a new format. This step could take a variable amount of time that will depend
+on the performance of the hardware you are using.
+
+Other than that upgrading from a version lower than 1.3.0.0 your client is probably stuck
```
Comment (Greg-Griffith) [inline doc/release-notes/release-notes-bucash1.5.1.0-rc1.md]: update 1.3.0.0 to 1.5.0.1 since that was the first client supporting this round of HFs

Comment (sickpig) [reply doc/release-notes/release-notes-bucash1.5.1.0-rc1.md]: done

**Comment by sickpig** [comment]
> We should remove the downgrade section because downgrading to older clients will put you on minority/dead forks.

agreed. removed

File: doc/release-notes-bucash.md [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
+Bitcoin Unlimited Cash Edition release candidate 1 for version 1.5.1.0 is now available from:
 
   <https://bitcoinunlimited.info/download>
 
 Please report bugs using the issue tracker at github:
 
   <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
 
-This is a minor bugs fix only release version based of Bitcoin Unlimited compatible
+This is a main release candidate based of Bitcoin Unlimited compatible
```
Comment (awemany) [inline doc/release-notes-bucash.md]: As this just caught my eye: Don't we want to get rid of 'based' here?

Also I wonder about formatting wrt. line-lengths. It looks to me this could profit from a reformat to <80 lines.

File: doc/release-notes/release-notes-bucash1.5.1.0-rc1.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+Bitcoin Unlimited Cash Edition release candidate 1 for version 1.5.1.0 is now available from:
+
+  <https://bitcoinunlimited.info/download>
+
+Please report bugs using the issue tracker at github:
+
+  <https://github.com/BitcoinUnlimited/BitcoinUnlimited/issues>
+
+This is a main release candidate based of Bitcoin Unlimited compatible
```
Comment (awemany) [inline doc/release-notes/release-notes-bucash1.5.1.0-rc1.md]: same here

**Review by awemany — APPROVED** [review]
Looks good except for two more minor nits.

File: doc/release-notes-bucash.md (hunk 3 above)
Comment (sickpig) [reply doc/release-notes-bucash.md]: fixed.

WRT 80 chars long line: the render of the markdown file are independent from the actual len of a line, but I think you have a point. 

I will start enforcing a max line length starting from the next release :smile:

File: doc/release-notes/release-notes-bucash1.5.1.0-rc1.md (hunk 4 above)
Comment (sickpig) [reply doc/release-notes/release-notes-bucash1.5.1.0-rc1.md]: fixed

**Review by Greg-Griffith — APPROVED** [review]

**Review by awemany — APPROVED** [review]
Only nit left: Typo PR title :D
